src/vdaf/prio3.rs

```diff
@@ -239,6 +121,16 @@ where
     T: Type,
     P: Prg<L>,
 {
+    /// Constructor, because our fields are not public
```

Comment:
This doccomment will end up on docs.rs so let's try for a nicer explanation, consistent with the other `new_*` functions in the module:
```suggestion
    /// Construct an instance of this Prio3 VDAF with the given number of aggregators and the
    /// underlying type.
```